doc/access-logs.md

```diff
@@ -0,0 +1,14 @@
+## Understanding Access Logs
+
+### Duration fields
+
```

Comment:
I think it would be helpful if we included a diagram to help folks visualize this. Like this one you and @ndhanushkodi made! 😲

https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG4gICAgcGFydGljaXBhbnQgQ2xpZW50XG4gICAgcGFydGljaXBhbnQgRW52b3lcbiAgICBwYXJ0aWNpcGFudCBOZXR3b3JrXG4gICAgcGFydGljaXBhbnQgVXBzdHJlYW1cbiAgICBDbGllbnQgLT4-IEVudm95OiBkdXJhdGlvbiwgcmVzcG9uc2VfZHVyYXRpb25cbiAgICBFbnZveSAtPj4gTmV0d29yazogcmVzcG9uc2VfZHVyYXRpb24sIGR1cmF0aW9uXG4gICAgTmV0d29yayAtPj4gVXBzdHJlYW06IGR1cmF0aW9uLCByZXNwb25zZV9kdXJhdGlvblxuICAgIFxuICAgIFVwc3RyZWFtIC0-PiBOZXR3b3JrOiByZXNwb25zZV9kdXJhdGlvbiwgZHVyYXRpb25cbiAgICBOZXR3b3JrIC0-PiBFbnZveTogZHVyYXRpb24sIHJlc3BvbnNlX2R1cmF0aW9uLCByZXNwb25zZV90eF9kdXJhdGlvblxuICAgIEVudm95IC0-PiBDbGllbnQ6IGR1cmF0aW9uLCByZXNwb25zZV90eF9kdXJhdGlvbiIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19

Reply:
roger that :salute:

Reply:
@tcdowney fixed.

Reply:
Looks good, thank you! 📈